site/src/api/api.ts

```diff
@@ -282,7 +282,19 @@ export const suspendUser = async (userId: TypesGen.User["id"]): Promise<TypesGen
   return response.data
 }
 
-export const postFirstUser = async (
+export const hasFirstUser = async (): Promise<boolean> => {
+  try {
+    // This endpoint returns 404 if it is false or a 200 if it is success. You
+    // can see its definition here:
+    // https://github.com/coder/coder/blob/db665e7261f3c24a272ccec48233a3e276878239/coderd/users.go#L33-L53
+    await axios.get("/api/v2/users/first")
+    return true
+  } catch {
```

Comment:
If we don't check the specific error, is it possible that we show the setup page when an unrelated database error occurs?

Reply:
Good call, yes. I was just lazy on this one because I'm not sure how the FE could handle this but definitely checking the specific error is better for debugging later.

Reply:
Done!